examples/intkey_go/tests/test_intkey_smoke_go.yaml

```diff
@@ -38,6 +38,7 @@ services:
     image: hyperledger/sawtooth-validator
     expose:
       - 4004
+      - 5050
```

Comment:
In the future, I suggest making this kind of changes in their own pr. This will help get quick fixes like this in quicker, without requiring approval of all of the other changes and components.

Reply:
Sure, agree. It happened this time because original PR was raised when tests used older validator (without consensus SDK) and it was required to make build pass.